io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/NOTICE

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Do we need this?

Reply:
Removed the file in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/7bc0a7b510c5137c156f5317497b12e794ef3993